server/sonar-main/src/main/java/org/sonar/application/process/ProcessLauncherImpl.java

```diff
@@ -122,9 +116,7 @@ public ProcessMonitor launch(JavaCommand javaCommand) {
       ProcessCommands commands = allProcessesCommands.createAfterClean(processId.getIpcIndex());
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Use try-with-resources or close this "ProcessCommands" in a "finally" clause. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2095)